pkg/handlers/nutanix/mutation/controlplanefailuredomains/variables_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2025 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package controlplanefailuredomains
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/utils/ptr"
+
+	capxv1 "github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/api/external/github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
+	"github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/api/v1alpha1"
+	"github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/common/pkg/testutils/capitest"
+	nutanixclusterconfig "github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pkg/handlers/nutanix/clusterconfig"
+)
+
+func TestVariableValidation(t *testing.T) {
+	emptyFDs := minimumClusterConfigSpec()
+
+	validFDs := minimumClusterConfigSpec()
+	validFDs.ControlPlane.Nutanix.FailureDomains = []string{"fd-1", "fd-2", "fd-3"}
+
+	duplicateFDs := minimumClusterConfigSpec()
```

Comment:
Should there be a test case that uses this? I don't see one.

Reply:
removed the unused vars